src/Weather.js

```diff
@@ -0,0 +1,33 @@
+import { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Accordion from 'react-bootstrap/Accordion';
+
+class Weather extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      weatherData: this.props.weatherData,
+    }
+  }
+  render() {
+    let weatherArray = this.state.weatherData;
```

Comment:
Nice work so far!

I know you mentioned your weather data wasn't updating and so here's some food for thought:

-The weather data is being set in state at construction time and that value in state never changes (which requires a setState call)
- In a situation when you have a component whose state depends on props, `componentDidUpdate()` can be used to trigger state changes. Don't worry about this yet though because you should be able to build this component without state.